PhysicsTools/NanoAOD/python/fsrPhotons_cff.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.NanoAOD.common_cff import *
+
+from CommonTools.RecoUtils.leptonFSRProducer_cfi import leptonFSRProducer
+leptonFSRphotons = leptonFSRProducer.clone(
+  packedPFCandidates = cms.InputTag("packedPFCandidates"),
+  slimmedElectrons = cms.InputTag("slimmedElectrons"), #for footrprint veto
+  muons = cms.InputTag("linkedObjects","muons"),
+  electrons = cms.InputTag("linkedObjects","electrons"),
```

Comment:
Do not write explicit types when updating cloned parameters
```suggestion
  packedPFCandidates = "packedPFCandidates",
  slimmedElectrons = "slimmedElectrons", #for footrprint veto
  muons = "linkedObjects:muons",
  electrons = "linkedObjects:electrons",
```